apps/cowswap-frontend/src/modules/hooksStore/dapps/WithdrawCoWAMMApp/hook.tsx

```diff
@@ -0,0 +1,33 @@
+import { HookDappInternal, HookDappType } from 'modules/hooksStore/types/hooks'
+
+import withdrawImage from './icon.png'
+
+import { WithdrawHookApp } from './index'
+
+const Description = () => {
+  return (
+    <>
+      <p>
+        Reduce or withdraw liquidity from a pool before a token swap integrating the process{' '}
```

Comment:
I assume this is a CoW AMM pool, not any AMM pool right? would just indicate it in the copy